src/test/java/teammates/test/cases/action/InstructorFeedbackAddActionTest.java

```diff
@@ -219,11 +223,13 @@ protected InstructorFeedbackAddAction getAction(String... params) {
         return (InstructorFeedbackAddAction) gaeSimulation.getActionObject(getActionUri(), params);
     }
 
-    protected String getPageResultDestination(
-            String parentUri, String courseId, String fsname, String userId, boolean isError) {
+    protected String getPageResultDestination(String parentUri, String courseId, String fsname,
+            boolean isToBeLoadedInEditMode, String userId, boolean isError) {
```

Comment:
No need to modify `getInstructorFeedbackLink` in the action - this is only about the `getPageResultDestination` helper method in tests.